testing/run-scripts/run_cloud.sh

```diff
@@ -174,21 +177,14 @@ fi
 echo "CLOUD_INSTALL_STATUS_INSTALLING_SSH"
 echo "CLOUD_INSTALL_PROGRESS 60"
 if ! docker ps -a | grep uproxy-sshd >/dev/null; then
-  if ! docker images | grep uproxy/sshd >/dev/null; then
-    TMP_DIR=/tmp/uproxy-sshd
-    rm -fR $TMP_DIR
-    cp -R ${BASH_SOURCE%/*}/../../sshd/ $TMP_DIR
-
-    echo -n "$BANNER" > $TMP_DIR/banner
-    echo -n "$PUBLIC_IP" > $TMP_DIR/hostname
-
-    docker build -t uproxy/sshd $TMP_DIR
-  fi
-
   # Add an /etc/hosts entry to the Zork container.
   # Because the Zork container runs with --net=host, we can't use the
   # regular, ever-so-slightly-more-elegant Docker notation.
-  docker run --restart=always -d -p $SSHD_PORT:22 --name uproxy-sshd --add-host zork:$HOST_IP uproxy/sshd > /dev/null
+  docker run --restart=always -d -p $SSHD_PORT:22 --name uproxy-sshd --add-host zork:$HOST_IP $SSHD_IMAGE > /dev/null
+  docker exec uproxy-sshd sh -c "echo $BANNER > /banner"
+  docker exec uproxy-sshd sh -c "chmod 644 /banner; chown giver: /banner"
```

Comment:
Since `sh -c` is really just a trick to pipe input into the command for `echo`-ing, how about we make this line:
```bash
docker exec uproxy-sshd chmod 644 /banner
docker exec uproxy-sshd chown giver: /banner
```